x-pack/docs/en/security/operator-privileges/operator-only-snapshot-and-restore.asciidoc

```diff
@@ -0,0 +1,19 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[operator-only-snapshot-and-restore]]
+=== Operator Privileges and Snapshot/Restore
+The invocation of <<operator-only-apis>> and update of <<operator-only-dynamic-cluster-settings>>
+result changes in the cluster state, which can be <<snapshot-restore,snapshot and restored>>.
+Restoring snapshot data associated to <<operator-only-functionality,operator-only functionality>>
+is problematic because:
```

Comment:
I think we want to rethink this sentence. It's not problematic, because we made it do the right thing.

I think you want to restructure this whole section to say:
- We do this
- Because the infrastructure owners want this
- So we avoid this bad stuff

Reply:
You are right. It reads a bit negative for the restore process which isn't true. I reworded a bit and added some context.